src/configure

```diff
@@ -279,7 +283,7 @@ function linux_configure_omplibdir {
   for libext in $libexts; do
     echo "Testing $libext from [$libexts] " >&2
 
-    check_library $mkllibdir "libiomp5" $libext \
+    check_library $mkllibdir "libiomp5" $libext || check_library $mkllibdir "libmkl_tbb_thread" $libext  \
```

Comment:
Even this looks suspicious (because of the order of || and && -- I think the mixing of the operators is generally discouraged by  bash docs)